README.md

```diff
@@ -1,39 +1,61 @@
+- Automatic detection of common data formats
+- Schema inference during ingestion
+- Beautiful, intuitive result views for nested or tabular data
+- Named queries with version history
+- Query session history to keep track of your work
+- Pinnable query fragments to keep your search box uncluttered
+- Convenient right-click menus for pivoting and filtering
+- Exporting results to a file
 
-Brim is built from open source components, including:
+## Ready To Try It Out?
 
-- [Zed](https://github.com/brimdata/zed), a structured log query engine;
-- [Electron](https://www.electronjs.org/) and [React](https://reactjs.org/) for multi-platform UI;
-- [Zeek](https://www.zeek.org), to generate network analysis data from packet capture files.
+**[Download Zui](https://www.brimdata.io/download/)** for your operating system.
 
-## Installing Brim
+Refer to the [installation guide](https://github.com/brimdata/brim/wiki/Installation) and
+[releases notes](https://github.com/brimdata/brim/releases) for more information.
 
-See the [installation guide](https://github.com/brimdata/brim/wiki/Installation)
-in the [wiki](https://github.com/brimdata/brim/wiki). Release notes are
-available at the [releases](https://github.com/brimdata/brim/releases) page.
+## Powered By Zed
 
-## Having a problem?
+[Zed](https://zed.brimdata.io/docs) offers a fresh and innovative approach to working with data known as "[Super-Structured Data](https://www.brimdata.io/blog/super-structured-data/)".
+
+Behind Zui is a local Zed Lake instance where you can load your data into pools and use the powerful Zed language to search, analyze, and transform it. Use it to:
+
+- Explore deeply nested JSON objects
+- View parquet files
+- Clean up CSV files by adding type information
+- Search heterogeneous NDJSON logs
+- Transform data from a legacy database's CDC logs
+- Investigate Zeek security logs
+
+Zed has everything you need to make your work easier and more efficient. The [storage layer](https://zed.brimdata.io/docs/formats), [type system](https://zed.brimdata.io/docs/formats/zed), [query language](https://zed.brimdata.io/docs/language/overview), and [zq](https://zed.brimdata.io/docs/commands/zq) command-line tool are just a few of the tools and features that Zed offers to the data community.
+
+## Formerly Known as "Brim"
+
+For many years, the app was known as Brim, named after the company that created it [Brim Data](https://www.brimdata.io/). In 2023, it was renamed to Zui, a play on "Zed User Interface", to better reflect its connection to the Zed technology that powers it.
+
+Zui retains the security-specific features that made Brim popular while expanding it's reach to anyone who has data to work with. For example, you'll still find the customized views, histograms, and correlations relevant to the security domain appearing in Zui via our nacent plugin system. In the future, developers will be able to create custom plugins that make Zui even more effective for their specific needs.
```

Comment:
@nwt: I'm going to drop this suggestion of yours because I made another suggestion in the same line, but I included your it's->its fix in mine as well.